<?xml version="1.0"?>
<launch>

  <group ns="main_robot">
    <include file="$(find ak_eurobot_navigation)/launch/predicted_planning.launch"/>
    <!--<include file="$(find ak_eurobot_navigation)/launch/rrtstar.launch"/>-->

    <node name="motion_planner" pkg="ak_eurobot_navigation" type="motion_planner.py" respawn="true" output="screen">
      <rosparam file="$(find ak_eurobot_navigation)/config/motion_planner_main_robot_params.yaml" command="load" />
    </node>

  </group>

</launch>